infrastructure/log_analytics.bicep

```diff
@@ -10,3 +10,44 @@ resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2022-10
 }
 
 output id string = logAnalyticsWorkspace.id
+
+param alertRuleName string = 'Avg processing time'
+
+resource alertRuleAvgProcessingTime 'microsoft.insights/scheduledqueryrules@2023-03-15-preview' = {
+  name: alertRuleName
+  location: location
+  properties: {
+    displayName: alertRuleName
+    severity: 2
+    enabled: false
```

Comment:
Just a small note here. When the alert is created, and you try to look for it from the Monitor>Alert view. Is not initially visible as its disabled. Changing the Status filter will add it to the list. Just FYI in case someone was looking for it from that view like me. 
![Alert_rules_-_Microsoft_Azure](https://github.com/observability-lab-cse/observability-lab/assets/22852510/ac3a80c6-9553-42ce-8882-15aaadc2d1d2)